cassandra/writer.go

```diff
@@ -79,9 +138,12 @@ func (w *Writer) Write(tsb model.TimeSeriesBatch) error {
 }
 
 func (w *Writer) worker() {
+	w.workerCount.WithLabelValues("idle").Inc()
 	for m := range w.ch {
 		id := m.ts.ID()
 		for _, s := range m.ts.Samples {
+			w.workerCount.WithLabelValues("idle").Dec()
```

Comment:
Why did you update the goroutine here and not in the outer range when the message was first received off the channel?